python/stemwords.py

```diff
@@ -81,7 +81,7 @@ def stemming(lang, input, output, encoding, pretty):
     for original in codecs.open(input, "r", encoding).readlines():
         original = original.strip()
         # Convert only ASCII-letters to lowercase, to match C behavior
-        original = ''.join((lower_(c) if 'A' <= c <= 'Z' else c for c in original))
+        original = ''.join((c.lower() if 'A' <= c <= 'Z' else c for c in original))
```

Comment:
I was curious where this came from and if it ever worked so looked back to where it was introduced.  It was added in 2015 (e7e180d8561af193aeb777115ecea6a539080d9f) by you it seems!  I didn't find an answer as to whether it worked back then - the testsuite doesn't pass upper-case ASCII so quite possibly not.

Reply:
Indeed it was my commit, but I just imported @shibukawa's code so this originated there. Maybe it was copied from some other language than Python.